agent/consul/autopilot.go

```diff
@@ -68,73 +68,74 @@ func (s *Server) autopilotLoop() {
 			}
 
 			if err := s.pruneDeadServers(autopilotConfig); err != nil {
-				s.logger.Printf("[ERR] autopilot: error checking for dead servers to remove: %s", err)
+				s.logger.Printf("[ERR] autopilot: Error checking for dead servers to remove: %s", err)
 			}
 		}
 	}
 }
 
+// fmtServer prints info about a server in a standard way for logging.
```

Comment:
Sorry I didn't split this out - made some small logging fixes while I was in here.